docs/design/easydl.md

```diff
@@ -0,0 +1,109 @@
+# EasyDL Design
+
+EasyDL is an auto-configuration system for parameter server based training jobs.
+With EasyDL system, users need not provide any resource configuration for their 
+deep learning training jobs. Instead, the EasyDL can pick up the appropriate resource 
+configuration for each job smartly and continue to optimize those jobs during their runtime.
+
+## Background
+
+In parameter server based jobs, all relevant parameters of the training model 
+are distributed on the parameter server nodes. Each worker node takes partial training data
+as input and compute the new parameters. After that, the worker node sends the
+updated parameters to the parameter server node which is keeping the parameters. 
```

Comment:
"the updated parameters" -> "update parameters"
"the parameters " -> "parameters"